fiftyone/server/view.py

```diff
@@ -161,7 +158,7 @@ def get_saved_view_stages(identifier):
     return list(
         db.views.aggregate(
             [
-                {"$match": {"slug": identifier}},
+                {"$match": {"name": identifier}},
```

Comment:
the `name` is only unique relative to a dataset. to identify a specific view, either have to use `_id` or `name`+ `_dataset_id`

Reply:
I don't think the function is used anywhere. Can I remove?

Reply:
yeah it's no longer used because directly querying the db like this isn't more efficient that iterating on the saved views property on the dataset singleton. You can remove it.